plan.md

```diff
@@ -1,9 +1,282 @@
+
+## Branches
+All branches **must contribute to our fixed overheads** and/or contribute to our
+credibility as experts in a given field.
+
+Each branch will have **its own set of goals**, derived from [dwyl's guiding principles](github.com/dwyl/phase-two#guiding-principles)
+which will be refined over time to serve as it's _decision making framework_.
+
+> There is overlap across all of our branches - they all present multiple opportunities to feed into each other.
+
+### Product
+Building products is how we serve the _most_ people.
+
+Our intention has always been to be a 'products company'.  
+Building tools that make *our* lives easier and more effective as an
+organisation means that we will be start by solving a problem we not only already
+intimately familiar with, but are passionate about solving.
+
+Market research will be carried out and a business case will be put together for
+the creation of each of our products.
+
+#### Goals
++ Do we use/reach for the product _every day_?
++ Further goals to be set:
+  + Date for MVP
+  + Number of _active_ people by the end of the calendar year (based on repeated interaction with the app, [not time spent on it](https://github.com/dwyl/hq/issues/149#issuecomment-269154087))
+
+#### What?
+
+#### First products
+> Our **first products** will be [time](github.com/dwyl/time) (name TBC!) and [tudo](github.com/dwyl/tudo).
+
+Built with a modular architecture, this application will be a cross between a
+to-do list and a time tracking application.
+
+Our products will benefit from a positive [network effect](https://medium.com/evergreen-business-weekly/the-power-of-network-effects-why-they-make-such-valuable-companies-and-how-to-harness-them-5d3fbc3659f8#.6w6z2hjew) where increased usage of the product leads to it being a more valuable product for everyone (plays into open source).
+
+#### Key aims:
++ Effortlessness (e.g. one-click timer start)
++ Minimum friction (e.g. If a GitHub URL is added as a task, all of the data is pulled from GitHub and filled in automatically)
++ Improved focussed time
++ Effective time management (understand where your time is going & how to improve)
++ Improved team communication
++ Focus on the individual's potential/productivity first
++ Ability to share with the team built in from the beginning so people know
+when they would be interrupting ("I'm on fire" one-click warning to others
+:wink:) - key is to do a lot of research around what people _want_ to share
+and what they want to be shared _with them_
+
+#### CRM
+We have outgrown our 'spreadsheet' customer relationship management (CRM) system
+and need a place where we can **securely and accessibly** store all our client
+communications so that the *whole team* can see and contribute to these.
+
+Our CRM *code* will be **fully open source**, following our [guiding principles](https://github.com/dwyl/phase-two#guiding-principles) and
```

Comment:
same `404` link again:
![image](https://cloud.githubusercontent.com/assets/194400/23900991/366a69b0-08b3-11e7-9ad0-4519424113a9.png)

did you mean `/README.md#guiding-principles` ?